src/dolphin/filtering.py

```diff
@@ -0,0 +1,198 @@
+import numpy as np
+from numpy.typing import ArrayLike
+from scipy import ndimage
+
+
+def filtering(
+    unw_ifg: ArrayLike,
+    corr: ArrayLike,
+    mask_cutoff: float = 0.5,
+    wavelength_cutoff: float = 50 * 1e3,
+    dx: float = 30,
+) -> np.ndarray:
+    """Filter out signals with spatial wavelength longer than a threshold.
+
+    Parameters
+    ----------
+    unw_ifg : np.ndarray, 2D complex array
+        unwrapped interferogram to interpolate
+    corr : 2D float array
+        Array of interferometric correlation from 0 to 1.
+    mask_cutoff: float
+        Threshold to use on `corr` so that pixels where
+        `corr[i, j] > mask_cutoff` are True and the rest are False.
+        The default is 0.5.
```

Comment:
```suggestion
        Threshold to use on `correlation` so that pixels where
        `correlation[i, j] > mask_cutoff` are used and the rest are ignored.
        The default is 0.5.
```